department-of-veterans-affairs/vets-api
# [VI-691] Move MHV account creation to UserLoader

## Summary
-  Move MHV account creation call to UserLoader.
- Remove overall after login gate since we have a flipper

## Related issue(s)

- https://jira.devops.va.gov/browse/VI-691

## Testing 
- Enable flipper - `Flipper.enable(:mhv_account_creation_after_login)`
-  Login with SiS
- Check that sidekiq jobs are getting enqueued - http://localhost:3000/sidekiq/queues
- check that jobs are not enqueued when its disabled
